tables/iRO/Restart/shuffle.txt

```diff
@@ -1,218 +1,54 @@
-# Generated: 2018-07-19T07:53:04Z
-007D 089E
-0090 08A4
-00B8 0889
-0126 096A
-0127 0926
-0140 08AB
-0178 0862
+# Generated: 2018-08-09T00:00:50Z
+007D 0365
+0090 0A5A
+00B8 0361
+0126 0140
+0127 0364
+0140 07EC
+0178 014D
 0197 0004
```

Comment:
0004 - can you re-check this one, the recvpacket is weird , both restart and renewal